examples/01_example_wsiread.ipynb

```diff
@@ -21,20 +21,40 @@
     "id": "p3l08vukKAWo"
```

Comment:
<p><u>The next cell installs tiatoolbox including any pre-requisite software.</u></p><p>The next cell installs tiatoolbox, including prerequisite software, oni Colab or Kaggle, but should be skipped on other platforms.</p><p><em>If you accept my suggestions, the documentation is now repetitive, and the repetition should be eliminated.</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/0a7e3eef1bcb98b1137180859fd0cdeb8769d9e9